.github/workflows/python.yml

```diff
@@ -28,3 +28,7 @@ jobs:
         shell: bash
         run: |
           python --version
+          python setup.py install
+          pip install pytest
```

Comment:
pending: move to requirements.txt under tests/

Reply:
Hey quick question. I've written all the unit tests using unittest rather than pytest. Do we need pytest for this to work?

Reply:
both should work, where are those unit tests? are they in the repo?

Reply:
let me know and we can wire those up here and I can remove the pytest and my dummy test that I added

Reply:
oh I just saw them in this repo